config/sync/shortcut.set.default.yml

```diff
@@ -1,4 +1,4 @@
-uuid: null
+uuid: e550f64a-ef17-49e1-9cb8-b08a8421ff37
```

Comment:
We never want to commit changes to shortcut.set.default.yml, it causes an error when installing the site